src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -336,7 +350,11 @@ export const CloudEnvironmentModal = ({
     const app = currentApp(toolLabel)
     const doesCloudEnvForToolExist = currentRuntimeTool === toolLabel || app
     const isCloudEnvForToolDisabled = isCloudEnvModalDisabled(toolLabel)
+    // for Public Preview disable the option to create a new Cromwell app in Azure
+    const isSettingsOptionDisabled = toolLabel === toolLabels.CromwellOnAzure
```

Comment:
A better way to do this would be to push it into `CromwellOnAzure` itself, like `isPauseUnsupported`.

Reply:
Or could you just do this through the existing `isHidden` method?

Reply:
Good point. I will add it to `CromwellOnAzure` 👍